ddp-workspace/projects/ddp-mbc/src/assets/i18n/es.json

```diff
@@ -339,112 +339,113 @@
                 "Section1": {
-                    "Pt1": "[es]The Metastatic Breast Cancer Project takes a new approach to cancer research in which researchers partner directly with patients, who share their samples and clinical information in order to speed important discoveries. The MBCproject is part of",
-                    "Pt2": "[es]a nonprofit organization that brings together patients and researchers as partners to accelerate discoveries in cancer research.",
-                    "Pt3": "[es]is stewarded by four leading organizations: Emerson Collective, a California-based social change organization; the Broad Institute of MIT and Harvard, a leading nonprofit biomedical research institution; the Biden Cancer Initiative, an independent nonprofit organization that builds on the federal government’s Cancer Moonshot; and the Dana-Farber Cancer Institute, a leading cancer hospital."
+                    "Pt1": "El Metastatic Breast Cancer Project es un nuevo modelo en la investigación del cáncer en el cual los investigadores colaboran directamente con pacientes, quienes comparten sus muestras de tumor e información clínica para juntos acelerar descubrimientos. El MBCproject es parte de",
+                    "Pt2": "una organización sin fines de lucro que atrae a pacientes e investigadores para acelerar descubrimientos en la investigación del cáncer.",
+                    "Pt3": "es codirigido por cuatro organizaciones líderes: Emerson Collective, una organización de transformación social ubicada en California; el Instituto Broad de MIT y Harvard (Broad Institute of MIT and Harvard), una institución de investigación biomédica sin fines de lucro; el Biden Cancer Initiative, una organización independiente sin fines de lucro que se basa en el Cáncer Moonshot del gobierno federal; y el Instituto de Cáncer Dana-Farber (Dana-Farber Cancer Institute), un hospital líder en el tratamiento del cáncer."
                 },
-                "Section2": "[es]Our goal is to reach out to patients with metastatic breast cancer throughout the USA and Canada in order to study their cancer using cutting-edge genomic and molecular research studies performed at the Broad Institute by allowing patients to participate in these studies regardless of where they live. We are also asking patients from all over the world to sign up so that we can collect information worldwide and eventually expand to additional countries."
+                "Section2": "Nuestra meta es darles a pacientes alrededor de los Estados Unidos y sus territorios y Canadá con cáncer de mama metastásico la oportunidad de participar en estos estudios sin importar donde vivan para poder analizar su cáncer usando estudios moleculares y genómicos de última generación llevados a cabo en el Instituto Broad (Broad Institute)."
             },
             "HowProjectWorks": {
-                "Title": "[es]Here’s how the project works:",
+                "Title": "Cómo Funciona el Proyecto:",
```

Comment:
Do we need keep every word capitalized  there?

Reply:
hmm, good catch! copied from the file, but I guess we don't need to capitalize it